source/platforms/snowflake.md

```diff
@@ -0,0 +1,343 @@
+
+CREATE OR REPLACE WAREHOUSE CONTAINER_HOL_WH
+  WAREHOUSE_SIZE = XSMALL
+  AUTO_SUSPEND = 120
+  AUTO_RESUME = TRUE;
+
+CREATE STAGE IF NOT EXISTS specs
+ENCRYPTION = (TYPE='SNOWFLAKE_SSE');
+
+CREATE STAGE IF NOT EXISTS volumes
+ENCRYPTION = (TYPE='SNOWFLAKE_SSE')
+DIRECTORY = (ENABLE = TRUE);
+```
+
+Then we proceed to create the external access integration, the compute pool (with
+GPU resources), and the image repository:
+
+```sql
+USE ROLE ACCOUNTADMIN;
+
+CREATE OR REPLACE NETWORK RULE ALLOW_ALL_RULE
+  TYPE = 'HOST_PORT'
+  MODE = 'EGRESS'
+  VALUE_LIST= ('0.0.0.0:443', '0.0.0.0:80');
+
+CREATE EXTERNAL ACCESS INTEGRATION ALLOW_ALL_EAI
+  ALLOWED_NETWORK_RULES = (ALLOW_ALL_RULE)
+  ENABLED = true;
+
+GRANT USAGE ON INTEGRATION ALLOW_ALL_EAI TO ROLE CONTAINER_USER_ROLE;
+
+USE ROLE CONTAINER_USER_ROLE;
+CREATE COMPUTE POOL IF NOT EXISTS CONTAINER_HOL_POOL
+MIN_NODES = 1
+MAX_NODES = 1
+INSTANCE_FAMILY = GPU_NV_S; -- instance with GPU
+
+CREATE IMAGE REPOSITORY CONTAINER_HOL_DB.PUBLIC.IMAGE_REPO;
+
+SHOW IMAGE REPOSITORIES IN SCHEMA CONTAINER_HOL_DB.PUBLIC;
+```
+
+## Docker image push via SnowCLI
+
+The nest step in the process is to push to the image registry the docker image
+you will want to run via the service.
+
+### Build Docker image locally
+
+For this guide, we build an image that starts from the RAPIDS notebook image and
+adds some extra snowflake packages.
+
+Create a Dockerfile as follow:
+
+```Dockerfile
+FROM rapidsai/notebooks:25.02a-cuda11.8-py3.11-amd64
+
+RUN pip install "snowflake-snowpark-python[pandas]" snowflake-connector-python
+```
+
+```{note}
+- The `python=3.11`, is the latest supported by the Snowflake connector package.
+- The use of `amd64` platform is required by Snowflake.
+```
+
+In the directory where your Dockerfile is located build the image. Notice that
+no GPU is needed to build this image.
+
+```bash
+docker build --platform=linux/amd64 -t <local_repository>/rapids-nb-snowflake:latest .
+```
+
+### Create local environment with SnowCLI
+
+Create a conda/mamba environment with SnowCLI.
+
+```yaml
+name: snow-cli
+channels:
+  - condaforge
+dependencies:
+  - python=3.11
+  - pip
+  - pip:
+      - snowflake-cli
+```
+
+Activate the environment `mamba activate snow-cli` and configure your Snowflake
+CLI connection
+
+```bash
+snow connection add
+```
+
+Follow the wizard.
+
+```bash
+connection name : CONTAINER_HOL
+account : <ORG>-<ACCOUNT-NAME> # e.g. MYORGANIZATION-MYACCOUNT
+user : <snowflake_user_name>
+password : <snowflake_password>
+role: CONTAINER_USER_ROLE
+warehouse : CONTAINER_HOL_WH
+database : CONTAINER_HOL_DB
+schema : public
+host:
+port:
+region:
+authenticator:
+private key file:
+token file path:
+```
+
+```{note}
+If you don't recall `<ORG>-<ACCOUNT-NAME>` you can obtain them
+by running the following in the Snowflake SQL worksheet.
+```
+
+```sql
+SELECT CURRENT_ORGANIZATION_NAME(); --org
+SELECT CURRENT_ACCOUNT_NAME();      --account name
+```
```

Comment:
I didn't see this until after I spent a while trying to figure out where to find this info. Maybe move this up, or mention it in the comment next to `account : <ORG>-<ACCOUNT-NAME> # e.g. MYORGANIZATION-MYACCOUNT`